Week01/Problem03/klai_03.py

```diff
@@ -0,0 +1,29 @@
+import math
+
+def findpFactor(n):
+  if n == 1:
+    return -1
+  for i in range(2,(int)(math.floor(math.sqrt(n)))+1):

```

Comment:
I'm surprised that calling this function with `600851475143` doesn't break your computer...the problem with Python 2 (which you must be using, otherwise the script wouldn't run) is that the `range(n)` function actually constructs a list `[0, 1, 2, ..., n]` in memory.  Even when reducing the size of the integer by taking `n = (n // (n**0.5))+ 1`, I'm still surprised that your computed has enough memory to construct that list.